### DISCUSSION TITLE
unknown error when playing music

### DISCUSSION THREAD

**Discussion by ketibanmeteor** [z-huang/music#1749] [Q&A]
![Screenshot_20241217-160025_InnerTune](https://github.com/user-attachments/assets/8b6309e1-a1fc-4348-8daa-10601123fc72)

**Comment by dopeternet** [comment]
same here,its down

**Reply by Daniel-1611** [reply]
Hello, the same error (unknown error) when I try to listen to non-downloaded music

**Comment by RougeParrot** [comment]
It's been showing "Unknown Error" in INDIA, onwards 4am. 
Right now it's 6.30am.
I tried VPN, reinstall, changing email, and also previous version. 
None worked. 
I'm in for patience because on how good this app is. 
Please resolve it asap. 
Thank you!

**Reply by karan657dot** [reply]
Even am also experiencing this issue since last night and have tried all possible methods to fix it till 6 am of today morning.we all know that how good this app is so please solve this issue ( unknown error) .so that we can experience and listen to music flowless.

**Reply by Isac-Coder** [reply]
En colombia acaba de pasar lo mismo no se a que hora empezo pero son las 4:24 del 18/12

**Reply by LambdaLekter** [reply]
Same here, I'm in Italy at 12.30 CEST

**Reply by RougeParrot** [reply]
https://www.reddit.com/r/fossdroid/comments/1hgwd48/innertune_not_working_temporary_fix/?utm_source=share&utm_medium=mweb3x&utm_name=mweb3xcss&utm_term=1&utm_content=share_button

**Comment by RougeParrot** [comment]
For a TEMPORARY FIX, visit my reddit page by searching "INNERTUNE (NOT WORKING) TEMPORARY FIX" -SOUL_BOXER. 
I have given a link out there for a temporal 'app.zip', which works just the fine for the time being.

**Reply by RougeParrot** [reply]
https://www.reddit.com/r/fossdroid/comments/1hgwd48/innertune_not_working_temporary_fix/?utm_source=share&utm_medium=mweb3x&utm_name=mweb3xcss&utm_term=1&utm_content=share_button

**Reply by niokis** [reply]
Thank you so much rouge,As for temporary help, it's quite useful. You're a genius.

**Reply by niokis** [reply]
Thank you so much rouge,As for temporary help, it's quite useful. You're a genius.

**Comment by Isac-Coder** [comment]
I have a question, is the OuterTune app a version of InerTune?
